packages/services/schema/src/lib/federation-tag-extraction.ts

```diff
@@ -231,11 +231,7 @@ export function applyTagFilterToInaccessibleTransformOnSubgraphSchema(
         }),
       } as ObjectLikeNode;
 
-      if (
-        !rootTypeNames.has(node.name.value) &&
```

Comment:
The root type name check here is unnecessary. If someone defines a contract that excludes the root type that is not our problem to ignore...